src/marqo/tensor_search/tensor_search.py

```diff
@@ -2105,10 +2168,12 @@ def vectorise_multimodal_combination_field_structured(
         start_time = timer()
         text_vectors = []
         if len(text_content_to_vectorise) > 0:
+            
             with RequestMetricsStore.for_request().time(f"create_vectors"):
+                prefixed_text_content_to_vectorise = text_processor.prefix_text_chunks(text_content_to_vectorise, text_chunk_prefix)
```

Comment:
can we verify through tests that this does not add double prefix? Like text_content_to_vectorise has prefixes, then this adds another prefix?

Reply:
Afaik, we don't prepend prefixes to text_content_to_vectorise, whose content is taken from `field_content` is not modified in both `_add_documents_unstructured` in the code block below `# All the plain tensor/non-tensor fields are processed, now we process the multimodal fields` . We only add prefixing in code block `# C) field type is standard`. 

I did some tests on this yesterday and seem to have confirmed that no double prefixing happens. I can write a more visible/specific test though.

Reply:
Actually I think `test_prefix_multimodal` shows this already

Reply:
Yep alright, just confirming! Sounds good then.